### DISCUSSION TITLE
"Change your password" popup warning of a data breach

### DISCUSSION THREAD

**Discussion by momesana** [cypress-io/cypress#29384] [General]
We use a very simple password, namely 'password' in our CI testing environment i. e. when running cypress. The issue is that now chrome shows a "Change your password" popup which in turn leads to a few failing tests (those that use the third party realHover plugin) as it interferes with the hover action. Is there a way to suppress the popup programmatically or make cypress do it?

![Screenshot from 2024-04-23 00-59-43](https://github.com/cypress-io/cypress/assets/12494/e8cc4672-2bee-46a8-b7d0-185d1895ea2d)

**Comment by JsuisUnLama** [comment]
Also interested in an answer to this.
For those wondering, it does not even seem to be true, because this pops up for:
- i!P[*n8µf9%
- ^pM5µ%87
- yU)!$o3P9

**Comment by dooleyb1** [comment]
@momesana did you ever get around this or find a solution? Experiencing the same also.

**Reply by SimonFischer04** [reply]
@dooleyb1 create a 
before:browser:launch in your config with:
launchOptions.preferences.default.profile['password_manager_leak_detection'] = false
